rotkehlchen/tests/api/test_users.py

```diff
@@ -354,6 +354,68 @@ def test_user_creation_with_already_loggedin_user(rotkehlchen_api_server, userna
     )
 
 
+# @pytest.mark.parametrize('start_with_logged_in_user', [True]) (NEEDED?)
+def test_user_password_change(rotkehlchen_api_server, username, db_password):
```

Comment:
Btw this is an integration test. So it's kind of heavier, since it brings up all the api and queries it e.t.c.

Could also have some unit tests which are in general preferred as they are faster. So just in the DB tests, see that you can change the password.

But we do indeed need at least 1 integration test per endpoint so this test is good. Just letting you know of the differences they have in the codebase.